### ISSUE TITLE
Inconsistent module naming

### ISSUE THREAD

**Issue by melowe** [apache/incubator-tuweni#208]
Publishing artefacts using publishToMavenLocal reveals an issue with dependency naming. While the artefacts are published using only the short module name some dependencies are declared using the Tuweni prefix..  I've been forced to hack a resolutionStrategy to deal with this but its far from ideal. 

```    
resolutionStrategy.eachDependency {details ->
  if(details.requested.group == "org.apache.tuweni" && details.requested.name.startsWith("tuweni")) {
        def shortName = details.requested.name.replaceFirst("tuweni-","")
        details.useTarget("org.apache.tuweni:"+ shortName +":"+ details.requested.version)
      }
}```

**Comment by atoulme** [comment]
Which dependencies do you see displaying this issue? Can you give a reproduction path?

**Comment by atoulme** [comment]
This was fixed for 2.0.0.
